src/jwt.cpp

```diff
@@ -24,7 +24,7 @@
 static const String base64_chars =
```

Comment:
I don't think we want to pave over the fixes from Alvaro in master, remove this file